tests/integration/it_roles_test.py

```diff
@@ -9,20 +9,44 @@ def test_list(client):
     assert client.get("/roles/").status_code == HTTPStatus.OK
 
 
-def test_create(client):
-    response = client.post(url_for("role.create"), data={
-        "name": "owner"
-    })
-    assert response.location.endswith(url_for('role.list_roles'))
-    assert Role.query.count() == 1
-    assert Role.query.get(1).name == "owner"
+def test_create(client, db_session):
+    assert Role.query.count() == 0
+    created = None
+    try:
+        response = client.post(url_for("role.create"), data={
+            "name": "owner"
```

Comment:
@ekondrashev ``name``, ``owner`` of what exactly? I would say ``normal_user`` and ``john``-- I suppose the ``owner`` is the employee who has said role, or?

Reply:
@amihaiemil I'm not an author not sure what was original thought here, I'll change this to `John`